examples/text-sentiment/start_runtime.py

```diff
@@ -13,12 +13,14 @@
 # limitations under the License.
 
 # Standard
-from os import path
+from os import path, chdir
 import sys
 
 # First Party
 import alog
 
+chdir(path.dirname(__file__))
```

Comment:
I'm curious why this is needed. If this is patching around the fact that we are expecting some assets to be referenced by a relative path, I think we should fix the places where that's true rather than doing this.

Reply:
Ah, so the issue is specifically with the fact that [the default value for `local_models_dir`](https://github.com/caikit/caikit/blob/main/caikit/config/config.yml#L70) is relative and there's no reliable way to ensure that we know the absolute path. I think the better solution then is to overwrite the config default to be the deduced absolute path as follows:

```py
import caikit

models_directory = os.path.abspath(
    os.path.join(os.path.dirname(__file__), "models")
)
caikit.config.configure(
    config_dict={
        "merge_strategy": "merge",
        "runtime": {"local_models_dir": models_directory},
    }
)
```

Reply:
Made the changes (and learned about `configure()`) 👍